src/CMakeLists.txt

```diff
@@ -637,17 +637,32 @@ if (BUILD_APP)
         )
 
     if (MSVC)
-        if (NOT VCPKG_INSTALLED_DIR)
```

Comment:
none of that should run when vcpkg toolchain is used, so `VCPKG_INSTALLED_DIR` check should be kept.

Reply:
Why? For me, it just leaves the folder empty.

Reply:
take a look on this
https://github.com/microsoft/vcpkg/blob/036769f5d6da667391c9240bc7ca56e84cacd5b1/scripts/buildsystems/vcpkg.cmake#L48
and these 2 scripts, called in post-build stage
https://github.com/microsoft/vcpkg/blob/master/scripts/buildsystems/msbuild/applocal.ps1
https://github.com/microsoft/vcpkg/blob/master/ports/qt5-base/qtdeploy.ps1

maybe some problem with running powershell on your side causes it to fail

Reply:
> called in post-build stage

This PR is not about building chatterino, it's about installing chatterino. When building, the deployment script is run, but it needs to run at installation as well.

Reply:
oh, i blame my lack of cmake knowledge then

it seems there is experimental switch for cmake install support, maybe it will help with your PR: https://github.com/microsoft/vcpkg/blob/036769f5d6da667391c9240bc7ca56e84cacd5b1/scripts/buildsystems/vcpkg.cmake#L52

anyways, my issue is resolved if it wont affect building

Reply:
Not sure how this works on Windows, but on Linux plugins are stored inside a `plugins` directory
e.g. if I install `extra/qt5-imageformats` it installs a directory under `/usr/lib/qt/plugins/` like this:
```
/usr/lib/qt/plugins
├── imageformats
│   ├── libqgif.so
│   ├── libqicns.so
│   ├── libqico.so
│   ├── libqjp2.so
│   ├── libqjpeg.so
│   ├── libqmng.so
│   ├── libqpdf.so
│   ├── libqsvg.so
│   ├── libqtga.so
│   ├── libqtiff.so
│   ├── libqwbmp.so
│   └── libqwebp.so
```

Reply:
when building with vcpkg it also places all plugins inside /plugins. When Qt search for plugins it checks root executable folder and also Plugins path from qt.conf file, which defaults to /[plugins](https://doc.qt.io/qt-6/qt-conf.html#overriding-paths) when not set (like in chatterino case).
Imo to keep it consistent with vcpkg build, dont move plugins in this PR

Reply:
This seems like the problem described here https://github.com/microsoft/vcpkg/issues/7461#issuecomment-659612311. The linked fixes mostly manipulate [`qt.conf`](https://doc.qt.io/qt-5/qt-conf.html), which didn't really work for me.